.gitignore

```diff
@@ -47,3 +47,12 @@ com_crashlytics_export_strings.xml
 crashlytics.properties
 crashlytics-build.properties
 fabric.properties
+
+# Idea
+.idea
```

Comment:
Are you sure that we don' want to add idea projects? i mean, at some point this should be generic solution but for now we all depend on project configurations. IMO this should be removed.